documentation/faq.asciidoc

```diff
@@ -5,6 +5,7 @@
 :toc:
 :toc-title:
 :toc-placement: macro
+:confluent-platform-version: 5.3.1
```

Comment:
Shouldn't this go to the central config file?

Reply:
Good point, moved.